test/unit/modules/utils/cookie-spec.js

```diff
@@ -112,10 +112,14 @@ describe('Keen.utils.cookie', function() {
 
   });
 
-  describe('.enabled', function(){
+  describe('enabled property', function(){
 
-    it('should return a boolean value', function(){
-      assert.isBoolean(this.cookie.enabled());
+    it('should be a boolean value by default', function(){
+      assert.isBoolean(this.cookie.enabled);
```

Comment:
This original check was in place to make it possible for devs to determine if cookies were enabled or not on the browser. Perhaps we need to replace this with a manual check to avoid the breaking change? 

Here's an example I've found on a few SO Qs and posts:

```javascript
cookie.prototype.enabled = function(){
  var enabled = (navigator.cookieEnabled) ? true : false;
  if (typeof navigator.cookieEnabled === 'undefined' && !enabled) {
    document.cookie = 'testcookie';
    enabled = (document.cookie.indexOf('testcookie') != -1) ? true : false;
  }
  return enabled;
};
```